main/lxc/lxc.initd

```diff
@@ -51,15 +51,36 @@ checkconfig() {
 	fi
 }
 
+systemd_ctr() {
+	# required for lxc-console & services inside systemd containers
+	local cgroup=/sys/fs/cgroup/systemd cmd=$1
+	local mnt_opts='rw,nosuid,nodev,noexec,relatime,none,name=systemd'
+
+	case "$cmd" in
+		 mount) checkpath -d $cgroup
+			if ! mount | grep $cgroup 1>/dev/null; then
+				mount -t cgroup -o $mnt_opts cgroup $cgroup
+			fi
+			;;
+	       unmount) if mount | grep $cgroup 1>/dev/null; then
+				umount $cgroup
+			fi
+			;;
+	esac
+}
+
 _autostart() {
 	ebegin "$1 LXC containers"
 	shift
-	lxc-autostart --group "${LXC_GROUP:-onboot,}" "$@"
+	lxc-autostart --group "${lxc_group:-onboot,}" "$@"
```

Comment:
this change will break backwards compatibility. It would be nice to have something like:
````sh
: ${lxc_group:=$LXC_GROUP}
````
so that things will still work after upgrade in case `LXC_GROUP` was set